services/worker/deploy-script.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+check_docker_installation() {
+  sudo docker --version
+  EXIT_CODE=$?
+  if [[ $EXIT_CODE -ne 0 ]]; then
+    sudo apt-get install docker.io
+    sudo systemctl start docker
+    sudo systemctl enable docker
+  fi
+}
+
+check_docker_installation
+
+CONF_FILE_PATH="./worker-conf.json"
+TAG=${1-latest}
+IMAGE=raonismaneoto/arrebol-worker:$TAG
+ID=`jq .Id $CONF_FILE_PATH`
+ID=${ID%\"}
+ID=${ID#\"}
```

Comment:
Could you add comments explaining these lines? Or even separate into a function with a self-explanatory name.